src/components/MiniMap/MiniMap.tsx

```diff
@@ -25,16 +25,12 @@ const MiniMap: FunctionComponent<MiniMapProperties> = ({
   mapOption,
   setMapOption,
 }) => {
-  const stateCode = getStateCode(region);
-  const stateName = getStateName(region);
-  assert(stateName);
-  assert(stateCode);
-  const showState = stateCode !== MAP_FILTERS.DC;
-
   const onSelectCounty = () => {
     setMobileMenuOpen(false);
   };
 
+  const showState = region.fipsCode !== '11';
```

Comment:
maybe `region.fipsCode.startswith('11')` (or whatever it is in typescript haha) since there can be dc state and counties (though we should change this)